.github/workflows/verify-images.yml

```diff
@@ -8,13 +8,20 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        version: [8, 11, 15]
+        version-package: [8-jdk, 11-jdk, 15-jdk, 15-slim]
```

Comment:
https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-including-additional-values-into-combinations

```
version: [8,11,15]
package: [jdk]
include: 
 - version: 15
    package: [slim]
```

Reply:
Nice! I've included the JRE 8 verification using the same include syntax, which cleans up the build a bit more too.